src/analysis/retail/evoker/augmentation/constants.tsx

```diff
@@ -9,6 +9,8 @@ export const EBON_MIGHT_BASE_DURATION_MS = 10000;
 
 // Mastery aura extension
 export const TIMEWALKER_BASE_EXTENSION = 0.04;
+// Multiplier for Sands coefficient to get the extension coefficient
+export const TIMEWALKER_EXTENSION_MULTIPLIER = 0.5 / 0.34;
```

Comment:
So looking at the current implementations that interact with mastery, they are all related to Time Walker and not Shift Sands. So I think we should simply change the `masteryCoefficient` value in `SPECS.ts` to use Time Walker coefficient instead. Then there won't be any need for the `TIMEWALKER_EXTENSION_MULTIPLIER` constant.

I think for clarity sake we can make a const for each coefficient and just import the Time Walker one into `SPECS.ts`.
I doubt that Shifting Sands coefficient will actually be used in analysis.
```suggestion
// Mastery Versatility buff
export const SHIFTING_SANDS_MASTERY_COEFFICIENT = 0.34;
// Mastery aura extension
export const TIMEWALKER_BASE_EXTENSION = 0.04;
export const TIMEWALKER_MASTERY_COEFFICIENT = 0.5;
```